### ISSUE TITLE
Missing files in master

### ISSUE THREAD

**Issue by sinyaa** [kmaragon/SpatialIndex.NET#4]
Cloning master and trying to build. It looks like there are two files missing in the repo:

Error   1   Could not copy the file "\SpatialIndex\libspatialindex\bin\spatialindex.dll" because it was not found.  SpatialIndex.NET.Test

Error   2   Could not copy the file "\SpatialIndex\libspatialindex\bin\spatialindexd.dll" because it was not found. SpatialIndex.NET.Test

**Comment by kmaragon** [comment]
Yeah sorry. I intentionally skipped out on writing all of the scripts to submodule

https://github.com/libspatialindex/libspatialindex

And build it as an external dependency in MSBuild. It's a mild pain to set up for very little benefit, since it's possible to just do. This whole library just contains the .NET bindings to use that C++ project in the CLR,
